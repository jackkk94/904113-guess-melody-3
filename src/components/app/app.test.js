import React from 'react';
import renderer from 'react-test-renderer';
import App from './app';

const name = `Tom`;

it(`<App /> should render Tom`, () => {
  const tree = renderer
    .create(<App name = {name}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

