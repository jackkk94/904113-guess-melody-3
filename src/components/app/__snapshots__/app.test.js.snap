// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render Tom 1`] = `
<div>
  Hello 
  Tom
   !
</div>
`;
